### ISSUE TITLE
Possible banner for new Comp and Pen notification preference from VA Notify

### ISSUE THREAD

**Issue by bevnobev** [department-of-veterans-affairs/va.gov-team#77798]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: VA Notify
- **OCTO product owner:** Beverly Nelson
- **Product name:** VA Notify
- **Product manager:**  Julie Martin/Melanie Jones
- **Slack channel:** [va-notify-public](https://dsva.slack.com/archives/C010R6AUPHT)
- **Dedicated content writer on your team (if you have one):** 
- **Dedicated a11y specialist on your team (if you have one):**
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else:  `Possible banner or other "advertisement" on va.gov for a new notification preference`

### What's the nature of your initiative and desired outcomes?

`Compensation and Pension Business Owners are working with VA Notify to launch a feature where Veterans can opt-in to receive text messages about their pension payment dates.  In order to maximize the number of Veterans that opt-in, the Compensation and Pension team would like to "advertise" the opt-in on va.gov `

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`The notification will be launching in late April/May timeframe.  Any help making Veterans aware of this new feature in the first 3 months would be greatly appreciated`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact <name>](https://va.gov) `placeholder link`

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/79026
```

```[tasklist]
### CAIA IA tasks
```

**Comment by aprocik1** [comment]
@bevnobev, thanks for submitting this intake request. We can't use a sitewide banner for this update, but we can explore other options. Before we offer any recommendations, we need to know how much information you have to communicate. Are there lengthy instructions or lots of FAQs for this service, similar to the [VEText service](https://www.va.gov/resources/vetext-for-va-health-care-reminders-and-updates/)? Or are there only a couple sentences you need to communicate?

If you have content already drafted, it'd be helpful to share it. If not, that's okay! We just need a better sense of how much content we're designing for.

**Comment by bevnobev** [comment]
@aprocik1 We don't have content drafted yet, but I would imagine it would be a few sentences similar to "Opt-in to receive a text messages notification when your recurring pension payment has processed" with a link the notification setting page under their profile.  FYI - the notification setting page is attached (from staging) 

![notification preferences](https://github.com/department-of-veterans-affairs/va.gov-team/assets/74934207/3852e0bb-9d67-453c-91bf-8303396fea06)

**Comment by aprocik1** [comment]
Thanks, @bevnobev. This is really helpful!

Since this content is only relevant for Veterans receiving payments, we're thinking of adding a new Q&A to the [View your VA payment history](https://www.va.gov/va-payment-history/) page. The Q&A would read something like this:

> **Can I be notified every time I get a VA payment?**
> Yes. You can receive a text message notification about your disability and pension payments.
> 
> You'll need to...[instructions to opt-in]

We'll workshop the content together later. What do you think about placing it on the payment history page?

**Comment by bevnobev** [comment]
Thanks @aprocik1!  It was mentioned that there is a Global Floating footer and that this type of "announcement" is a good match for the footer.  Does a Global Floating footer ring a bell?

**Comment by bevnobev** [comment]
Oh!  I think placing something on the payment history page is great idea

**Comment by aprocik1** [comment]
@bevnobev, our content team has never heard of a global floating footer. Do you have an example you can share?

Let's move forward on updating the payment history page. We'll share a preview link and screenshot for you to review in the next few weeks. We'll likely follow up with some clarifying questions before then.

**Comment by bevnobev** [comment]
@aprocik1 Just chatted with Danielle, and I think you refer to it as sticky footer.  Since this feature is only related to payments, it's not necessary to have a global element.  

Agree to move forward on updating the payment history page.

Thanks!

**Comment by aprocik1** [comment]
@bevnobev, we've drafted the Q&A for the payment history page. Here's a screenshot:

<img width="353" alt="2024 03 29 VA payment notification Q A" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/142408200/72007641-6747-40aa-b4fa-3689e1bd50f2">

If you'd like to view the Q&A on the page, here's [the preview link](https://preview-prod.vfs.va.gov/preview?nodeId=714). Note: You have to be on the VA network to access previews. And links on previews don't work.

Please let us know if we need to correct any of the content in the Q&A. And please let us know when we should publish it. Thanks!

**Comment by bevnobev** [comment]
Thanks @aprocik1.  This is great!  I've sent this to the business owners to get feedback on content.  Let's coordinate on when to publish so that we publish after the notification is turned on cc: @mjones-oddball

**Comment by aprocik1** [comment]
@bevnobev, I'm following to see if you've landed on a launch date for these notifications. When should we plan to publish this new Q&A? Thanks!

**Comment by aprocik1** [comment]
@mjones-oddball, great to be connected on this. My email is anthony.procik@va.gov. Are you available to meet Wednesday 7/10 from 9-10 am ET or 12:30-2:30 pm ET?

**Comment by mjones-oddball** [comment]
@aprocik1 I'm so sorry I missed this github notification! I sent you an invite for 1:30 pm today, but if that doesn't work anymore just let me know and we could meet tomorrow or early next week.

**Comment by mjones-oddball** [comment]
Notes:

- We should clarify that this is a recurring monthly pension or disability payment only. There are other payment types that exist which do not have text messages yet.
-  There is an intricacy where the Veteran needs to opt in by the 10th of the given month to receive that month's text message. If they opt in after the 10th they will receive future texts from the following month on.

**Comment by mjones-oddball** [comment]
@aprocik1 thanks so much for meeting with Ksenia and I today! I'll keep an eye out for the content revisions and will follow up here once I know more about the preference go-live timing. Ideally we can launch this in the next couple of weeks :)

**Comment by aprocik1** [comment]
@mjones-oddball, it was nice meeting you two. I've confirmed that CAIA manages [this page](https://www.va.gov/va-payment-history/) in Drupal, which means we don't need to coordinate with another team to publish updates. And I've flagged our tentative timeline to our copyedit, so she's prepared to publish updates next week. Here's the updated content:

<img width="359" alt="2024 07 10 VA Notify recurring payment text notification" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/142408200/73c31d5e-af75-431a-a545-56f39261df9d">

Any and all feedback welcome. Thanks!

**Comment by mjones-oddball** [comment]
@aprocik1 Only 1 tiny adjustment request. For the note about the 10th of the month, we aren't pulling the opt ins until the 11th at 6 am ET which basically gives folx thru Hawaii time the opportunity to opt in by midnight on the 10th. So I think we should say If you opt in before or on the 10th you'll get your first text that month, otherwise if it's after the 10th you'll get it the following month.

**Comment by aprocik1** [comment]
@mjones-oddball, here's the most up-to-date content. Can you confirm the date you'd like this to be published on VA.gov? I'll let our copyeditor know. Thanks!

<img width="357" alt="2024 07 15 VA Notify recurring payment text notification" src="https://github.com/user-attachments/assets/7b6e6099-9d45-4aef-bf77-00a946571eac">

**Comment by mjones-oddball** [comment]
@aprocik1 The final draft is perfect! Thank you for making those edits. I just sent an email that Auth Exp can enable the notification setting today. Once they get back to me that it's done then you can publish your piece :)

**Comment by mjones-oddball** [comment]
@aprocik1 AE was speedy! The notification setting is LIVE. You're cleared to publish the Q&A when ready. Can you please let me know when this is done?

**Comment by aprocik1** [comment]
@mjones-oddball, we published this Q&A on Friday. Here's the live page: [View your VA payment history](https://www.va.gov/va-payment-history/)

I'm moving this ticket to validate. If there's nothing left for us to accomplish, I'll close it as complete. Thanks for you collaboration!

**Comment by mjones-oddball** [comment]
@aprocik1 Looks great, thank you! You can close this :)
